.github/workflows/scripts/utilities.js

```diff
@@ -0,0 +1,73 @@
+module.exports = {
+  /**
+   * Get deleted files from pull request
+   *
+   * @param {Object} obj - Object for parameters
+   * @param {string[]} obj.paths - Array of paths to look for deleted files
+   */
+  getDeletedFilesFromPR: async ({ github, context, paths }) => {
+    const {
+      issue: { number: issue_number },
+      repo: { owner, repo }
+    } = context;
+
+    const deletedFiles = await github.paginate(
+      'GET /repos/{owner}/{repo}/pulls/{pull_number}/files',
+      { owner, repo, pull_number: issue_number },
+      (response) =>
+        response.data
+          .filter((file) => file.status === 'removed')
+          .filter((file) =>
+            paths.some((path) => file.filename.startsWith(path))
+          )
+    );
+
+    console.log('Deleted file count: ', deletedFiles.length);
```

Comment:
This logs to the workflow?

Reply:
Yea, it logs it to the workflow, like here: https://github.com/aws-amplify/docs/actions/runs/5182834462/jobs/9340044377?pr=5537